src/ui/libs/DatalensChartkit/ChartKit/helpers/apply-hc-handlers.ts

```diff
@@ -234,6 +30,19 @@ export const fixPieTotals = (args: {data: GraphWidget}) => {
     });
 };
 
+function getActionParams(params: Record<string, string | string[]> = {}) {
```

Comment:
We have already functions in dashkit for getting actionParams from params, see `pickActionParamsFromParams`. And it has different formats to return (with prefix or not).